content/azure-pipelines.yml

```diff
@@ -31,10 +36,17 @@ stages:
               projectKey: "Etch.OrchardCore.ModuleBoilerplate"
               projectName: "Etch.OrchardCore.ModuleBoilerplate"
 
-          - task: tmarkovski.projectversionasvariable.versionintovariable.projectversionasvariable@1
-            displayName: "Get Project Version as variables from Etch.OrchardCore.ModuleBoilerplate.csproj"
+          - task: replacetokens@5
             inputs:
               path: ./Etch.OrchardCore.ModuleBoilerplate.csproj
```

Comment:
This should be deleted.

Reply:
I deleted the enitre line, was that what you wanted?